<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTextDockWidget</class>
 <widget class="QDockWidget" name="EditTextDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Text</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="itemTypeTabs">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="objectTab">
       <attribute name="title">
        <string>Object</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="arrowTab">
       <attribute name="title">
        <string>Arrow</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="setSelectedButton">
      <property name="text">
       <string>Set selected</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="unicodeShapeCatcherButton">
      <property name="text">
       <string>Symbol helper ðŸ¡•</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
